docs/content/en/docs/clustermgmt/observability/cluster-verify.md

```diff
@@ -1,122 +1,44 @@
 ---
 title: "Verify cluster"
 linkTitle: "Verify cluster"
-weight: 11
+weight: 3
 aliases:
     /docs/tasks/cluster/cluster-verify/
 date: 2017-01-05
 description: >
   How to verify an EKS Anywhere cluster is running properly
 ---
 
-**Verify an EKS Anywhere cluster from the target cluster**
-First, set the `KUBECONFIG` environment variable is set up for the target cluster (management or workload cluster) of choice:
+### Verify an EKS Anywhere cluster
+
+First, set the `KUBECONFIG` environment variable for your cluster:
 
 ```
 export KUBECONFIG=<path-to-cluster-kubeconfig>
 ```
 
-To verify that the cluster components are up and running, use the `kubectl` command to show that the pods are all running.
-There will be more or less pods depending on whether a management or workload cluster is targeted.
+To verify that the cluster components are up and running, use the `kubectl` command to show that the pods are all running. Below is the example but output may change as per the deployment. Please ensure that pods should be in running STATUS:
```

Comment:
This is still a little confusing. We need to run whatever command is most representative of the health of the cluster, this includes the output. If we only show the `kube-system` pods, is that a properly running cluster? If yes, this is fine, if no, I'd prefer to leave the more detailed output or comment on the key pods that need to be running for all EKS-A clusters